gui/styles/_wizard.styl

```diff
@@ -81,27 +84,83 @@ a.more-info
   color var(--slateGrey)
 
 
-.folder__selector
-  @extend $form-select select
-  width: 100%
-  font-size: 1.1em
-  line-height: 1em
-  margin 0
-  text-align left
-  text-decoration: none
+.folder__subtitle
+  margin-bottom 0
+
+.folder__config
+  margin-top 1rem
```

Comment:
May I ask suggest to use utility classes for margin and padding to avoid hard-coded values?